src/content/docs/apm/agents/net-agent/net-agent-api/net-agent-api.mdx

```diff
@@ -0,0 +1,2579 @@
+    ITransaction transaction = agent.CurrentTransaction; 
+    transaction.AcceptDistributedTracePayload(metadata.Value, TransportType.Queue);
+    ```
+
+    </Collapser>
+    <Collapser
+        id="CreateDistributedTracePayload"
+        title="CreateDistributedTracePayload (obsolete)"
+    >
+    <Callout variant="caution">
+    This API is not available in the .NET agent v9.0 or higher. Please use [`InsertDistributedTraceHeaders`](/docs/agents/net-agent/net-agent-api/itransaction/#insertdistributedtraceheaders) instead.
+    </Callout>
+
+    Creates a distributed trace payload for inclusion in an outgoing request to a downstream system.
+
+    ### Syntax
+
+    ```cs
+    IDistributedTracePayload CreateDistributedTracePayload()
+    ```
+
+    ### Returns
+
+    An object that implements `IDistributedTracePayload` which provides access to the distributed trace payload that was created.
+
+    ### Usage considerations
+
+    * [Distributed tracing must be enabled](/docs/agents/net-agent/configuration/net-agent-configuration#distributed_tracing).
+    * `CreateDistributedTracePayload` will be ignored if `AcceptDistributedTracePayload` has already been called for this transaction.
+
+    ### Example
+
+    ```cs
+    IAgent agent = NewRelic.Api.Agent.NewRelic.GetAgent();
+    ITransaction transaction = agent.CurrentTransaction;
+    IDistributedTracePayload payload = transaction.CreateDistributedTracePayload();
+    ```
+
+    </Collapser>
+    <Collapser
+        id="AddCustomAttribute"
+        title="AddCustomAttribute"
+    >
+    Adds contextual information about your application to the current transaction in the form of [attributes](/docs/using-new-relic/welcome-new-relic/get-started/glossary#attribute).
+
+    This method requires .NET agent version and .NET agent API [version 8.24.244.0](/docs/release-notes/agent-release-notes/net-release-notes/net-agent-8242440) or higher. It replaced the deprecated [`AddCustomParameter`](/docs/agents/net-agent/net-agent-api/add-custom-parameter).
+
+    ### Syntax
+
+    ```cs
+    ITransaction AddCustomAttribute(string key, object value)
+    ```
+
+    ### Parameters
+
+    <table>
+    <thead>
+        <tr>
+        <th>
+            Parameter
+        </th>
+
+        <th>
+            Description
+        </th>
+        </tr>
+    </thead>
+
+    <tbody>
+        <tr>
+        <td>
+            `key`
+
+            _string_
+        </td>
+
+        <td>
+            Identifies the information being reported. Also known as the name.
+
+            * Empty keys are not supported.
+            * Keys are limited to 255-bytes. Attributes with keys larger than 255-bytes will be ignored.
+        </td>
+        </tr>
+
+        <tr>
+        <td>
+            `value`
+
+            _object_
+        </td>
+
+        <td>
+            The value being reported.
+            
+            <DoNotTranslate>**Note**</DoNotTranslate>: `null` values will not be recorded.
+            
+            <table>
```

Comment:
This table-in-a-table looks pretty nightmarish in builds, we should find another way to format it
<img width="686" alt="image" src="https://github.com/newrelic/docs-website/assets/55203603/738542eb-67d1-4537-b770-111db290c8b9">